db/users.js

```diff
@@ -0,0 +1,126 @@
+const client = require("./client");
+const bcrypt = require("bcrypt");
+
+// createUser({ username, password })
+// hash the password before storing it to the database
+async function createUser({ username, password }) {
```

Comment:
we need to add more parameters here in relation to the data needed in the users table:

createUser({ username, password, full_name, email })

and add these into the array as well